Code/Editor/Settings.cpp

```diff
@@ -130,7 +130,7 @@ SEditorSettings::SEditorSettings()
     viewports.bAlwaysShowRadiuses = false;
     viewports.bSync2DViews = false;
     viewports.fDefaultAspectRatio = 800.0f / 600.0f;
-    viewports.fDefaultFov = DEG2RAD(60); // 60 degrees (to fit with current game)
+    
```

Comment:
nit: I see the default here was 60 but in the settings registry it's changed to 90, is that correct/intended?

Reply:
I think the default is 60 not 90.

Reply:
Ah okay cool then we should change this (and ensure it's in the correct units) in the setreg viewport settings code